frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx

```diff
@@ -72,14 +72,18 @@ export const FormEndPageContainer = (): JSX.Element | null => {
       />
     )
   }
+
+  // Feedback is not supported on MRF
+  const isHideFeedback = form.responseMode === FormResponseMode.Multirespondent
+  console.log(isHideFeedback)
```

Comment:
redundant console log. Also, `isFeedbackHidden` maybe? `isHideFeedback` reads weirdly ahaha

Reply:
Great catch -- thought I've removed the comments earlier 🤦 .

cd19eb895